src/Sylius/Bundle/PayumBundle/Model/PaymentConfig.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+* This file is part of the Sylius package.
+*
+* (c) Paweł Jędrzejewski
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Sylius\Bundle\PayumBundle\Model;
+
+use Payum\Core\Model\PaymentConfigInterface;
+
+class PaymentConfig implements PaymentConfigInterface
```

Comment:
@Sylius/core-team I am almost done with ability to configure payments gateways in the backend. The last thing is to provide a backend part for this model `PaymentConfig`. The form type must be this one `payum_payment_config ` see: https://github.com/Payum/PayumBundle/blob/master/Resources/config/form.xml#L16